parts/k8s/openshiftmasterscript.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash -x
```

Comment:
Just a consideration: should we put these files in `parts/openshift`?

Reply:
Makes sense to me - https://github.com/kargakis/acs-engine/issues/98 as a follow-up

Reply:
ACK